include/exadg/incompressible_navier_stokes/postprocessor/line_plot_calculation.cpp

```diff
@@ -179,16 +188,19 @@ LinePlotCalculator<dim, Number>::evaluate(VectorType const & velocity,
 
               // write data
               for(unsigned int d = 0; d < dim; ++d)
-                f << std::setw(precision + 8) << std::left << points[i][d];
-              f << std::setw(precision + 8) << std::left << solution_vector[i];
+                f << std::setw(precision + 8) << std::left
+                  << evaluation_points[point_offset + i][d];
+              f << std::setw(precision + 8) << std::left << pressure_values[point_offset + i];
               f << std::endl;
             }
             f.close();
           }
         }
-      } // loop over quantities
-    }   // loop over lines
-  }     // write_output == true
+
+        point_offset += n_points;
```

Comment:
If this is the only occurrence of ``n_points``, please write ``(*line)->n_points`` here.